qodana.yaml

```
@@ -0,0 +1,6 @@
+version: "1.0"
+linter: jetbrains/qodana-jvm:2024.3
+profile:
+  name: qodana.recommended
+include:
+  - name: CheckDependencyLicenses
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)